HousingRegisterApi/V1/UseCase/GetNovaletExportUseCase.cs

```diff
@@ -0,0 +1,46 @@
+using HousingRegisterApi.V1.Domain.FileExport;
+using HousingRegisterApi.V1.Gateways;
+using HousingRegisterApi.V1.Services;
+using HousingRegisterApi.V1.UseCase.Interfaces;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace HousingRegisterApi.V1.UseCase
+{
+    public class GetNovaletExportUseCase : IGetNovaletExportUseCase
+    {
+        private readonly ILogger _logger;
+        private readonly IApplicationApiGateway _gateway;
+        private readonly ICSVService _csvService;
+
+        public GetNovaletExportUseCase(
+            ILogger<GetNovaletExportUseCase> logger,
+            IApplicationApiGateway gateway,
+            ICSVService csvService)
+        {
+            _logger = logger;
+            _gateway = gateway;
+            _csvService = csvService;
+        }
+
+        public async Task<FileExportResult> Execute(Guid id)
+        {
+            var application = _gateway.GetApplicationById(id);
```

Comment:
Not sure why we would only want to export one application at a time? The export should contain all the applicants that need to bid on properties.

Reply:
yup, corrected.. i thought the export was per application.. now changed to an array of..